setup.py

```diff
@@ -116,7 +116,7 @@ def localversion_func(version: ScmVersion) -> str:
         "requests>=2.0.0",
         "tqdm>=4.0.0",
         "torch>=1.7.0",
-        "transformers>4.0,<5.0",
+        "transformers>4.0,<4.52.1",
```

Comment:
this needs to be `<4.52.0`, right?

Reply:
4.42.0 is a yanked version so it should resolve to 4.51.x 
I can make it hard requirement